src/FondOfSpryker/Client/CompanyType/Dependency/Client/CompanyTypeToZedRequestClientBridge.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace FondOfSpryker\Client\CompanyType\Dependency\Client;
+
+use Spryker\Client\ZedRequest\ZedRequestClientInterface;
+use Spryker\Shared\Kernel\Transfer\TransferInterface;
+
+class CompanyTypeToZedRequestClientBridge implements CompanyTypeToZedRequestClientInterface
+{
+    /**
+     * @var \Spryker\Client\ZedRequest\ZedRequestClientInterface
+     */
+    protected $zedRequestClient;
+
+    /**
+     * @param \Spryker\Client\ZedRequest\ZedRequestClientInterface $zedRequestClient
+     */
+    public function __construct(ZedRequestClientInterface $zedRequestClient)
+    {
+        $this->zedRequestClient = $zedRequestClient;
+    }
+
+    /**
+     * @param string $url
+     * @param \Spryker\Shared\Kernel\Transfer\TransferInterface $object
+     * @param array|null $requestOptions
+     *
+     * @return \Spryker\Shared\Kernel\Transfer\TransferInterface
+     */
+    public function call($url, TransferInterface $object, $requestOptions = null): TransferInterface
```

Comment:
`(string $url, TransferInterface $object, ?array $requestOptions = null)`
Would be better